content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -599,28 +599,21 @@ documentation](/docs/tutorials/clusters/apparmor/#podsecuritypolicy-annotations)
 
 ### Seccomp
 
-The use of seccomp profiles in pods can be controlled via annotations on the
-PodSecurityPolicy. Seccomp is an alpha feature in Kubernetes.
+`seccompProfile` contains two fields that specify the seccomp profile that will
+be used for the pod.
 
-**seccomp.security.alpha.kubernetes.io/defaultProfileName** - Annotation that
-specifies the default seccomp profile to apply to containers. Possible values
-are:
+**Type** - determines the seccomp profile type for the pod
 
-- `unconfined` - Seccomp is not applied to the container processes (this is the
+- `Unconfined` - Seccomp is not applied to the container processes (this is the
   default in Kubernetes), if no alternative is provided.
-- `runtime/default` - The default container runtime profile is used.
-- `docker/default` - The Docker default seccomp profile is used. Deprecated as of
-  Kubernetes 1.11. Use `runtime/default` instead.
-- `localhost/<path>` - Specify a profile as a file on the node located at
+- `RuntimeDefault` - The default container runtime profile is used.
+- `Localhost` - Specify a seccomp profile as a file on the node. Must set
+  `LocalhostProfile` if this option is provided.
+
+**LocalhostProfile** - Path of the seccomp profile in the format
   `<seccomp_root>/<path>`, where `<seccomp_root>` is defined via the
   `--seccomp-profile-root` flag on the Kubelet.
```

Comment:
It may be worth mentioning that the flag `--seccomp-profile-root` is now deprecated and is due to be removed on v1.23.